doc/rtd/reference/datasources/lxd.rst

```diff
@@ -37,25 +36,28 @@ launched container, use either LXD profiles or
 ``lxc launch ... -c <key>="<value>"`` at initial container launch, by setting
 one of the following keys:
 
-- ``user.meta-data``: YAML metadata which will be appended to base metadata.
-- ``user.vendor-data``: YAML which overrides any metadata values.
-- ``user.network-config``: YAML representing either :ref:`network_config_v1` or
-  :ref:`network_config_v2` format.
-- ``user.user-data``: YAML which takes precedence and overrides both metadata
-  and vendor data values.
-- ``user.any-key``: Custom user configuration key and value pairs, which can be
-  passed to ``cloud-init``. Those keys/values will be present in instance-data
-  which can be used by both `#template: jinja` #cloud-config templates and
-  the :command:`cloud-init query` command.
+- ``cloud-init.vendor-data``: YAML which overrides any metadata values.
+- ``cloud-init.network-config``: YAML representing either
+  :ref:`network_config_v1` or :ref:`network_config_v2` format.
+- ``cloud-init.user-data``: YAML which takes precedence and overrides both
+  metadata and vendor data values.
+- ``user.<any-key>``: Keys prefixed with ``user.`` are included in
+  :ref:`instance data<instance_metadata>`. under the ``ds.config`` key. These
+  key-value pairs are used in jinja-templated :ref:`cloud-config<jinja-config>`
```

Comment:
docs CI test doesn't like this line:
`doc/rtd/reference/datasources/lxd.rst:45: : Spell check: templated: key-value pairs are used in jinja-templated .`